tests/system/provisioning/one_manager_agent/vars/configurations.yml

```diff
@@ -0,0 +1,11 @@
+cluster_key: "00000000000000000000000000000000"
+
+manager_hostname: "wazuh-manager"
+
+agent1_id: "001"
+agent1_hostname: "wazuh-agent1"
+agent1_key: "1111111111111111111111111111111111111111111111111111111111111111"
+
+docker_network: "cluster_net"
+
+image: "dontpanicat/debian:buster"
```

Comment:
Do not use third party docker images

Reply:
When we applied the suggested changes, we note that there is a conflict with access to the remote host via ssh. We will open an Issue to work on that.